delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/projects/model/ProjectHealer.scala

```diff
@@ -0,0 +1,40 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.projects.model
+
+import cats.effect.IO
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.delta.sdk.ScopeInitializer
+import ch.epfl.bluebrain.nexus.delta.sdk.identities.model.ServiceAccount
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.ScopeInitializationErrorStore
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.model.ProjectRejection.{ProjectHealingFailed, ProjectInitializationFailed}
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.ProjectRef
+
+trait ProjectHealer {
+
+  /**
+    * Heal the project
+    */
+  def heal(project: ProjectRef): IO[Unit]
+
+}
+
+object ProjectHealer {
+
+  def apply(
+      errorStore: ScopeInitializationErrorStore,
+      scopeInitializer: ScopeInitializer,
+      serviceAccount: ServiceAccount
+  ): ProjectHealer =
+    new ProjectHealer {
+      implicit private val serviceAccountSubject: Subject = serviceAccount.subject
+
+      override def heal(project: ProjectRef): IO[Unit] =
+        reinitializeProject(project) >> errorStore.delete(project)
+
+      private def reinitializeProject(project: ProjectRef): IO[Unit] =
+        scopeInitializer
+          .initializeProject(project)
+          .adaptError { case ProjectInitializationFailed(failure) => ProjectHealingFailed(failure, project) }
```

Comment:
We could keep the `ProjectInitializationFailed` error no ?

Reply:
The reason I didn't use it is because the message says `The project has been successfully created but it could not be initialized correctly` but you are not trying to create the project when using the healing endpoint. Also it returns a BadRequest instead of a 500 (but this might just be something that was forgotten).

I can still change it, just let me know what you think about making it a 500?